src/test/java/uk/gov/pay/ledger/pact/ConsumerContractTestSuite.java

```diff
@@ -5,6 +5,7 @@
 import junit.framework.TestSuite;
 import org.junit.runner.RunWith;
 import org.junit.runners.AllTests;
+import uk.gov.pay.ledger.pact.event.*;
```

Comment:
What's the reason for adding this line? Also we are for explicit importing for classes and avoid importing with `.*`.

Reply:
We import every class in this package, hence *

Reply:
That may be true but it doesn't address what our coding standards/style might be. I suggest asking the wider dev team for input here.

Reply:
I have replaced the wild card with individual imports.